package.json

```diff
@@ -91,20 +90,15 @@
     "electron-log": "^4.4.8",
     "electron-unhandled": "^3.0.2",
     "moment": "^2.29.4",
-    "playwright": "git+https://github.com/elastic/playwright.git#separate-recorder",
+    "playwright": "https://gitpkg.now.sh/elastic/playwright/packages/playwright-core?1.20.1-recorder",
     "react": "^17.0.1",
     "react-dom": "^17.0.1",
     "styled-components": "^5.3.5"
   },
-  "resolutions": {
-    "trim": "0.0.3",
-    "caniuse-lite": "1.0.30001359"
-  },
   "browserslist": [
     "last 1 version"
   ],
   "engines": {
-    "node": ">14.14.0",
-    "npm": "6"
+    "node": ">=16.15.0"
```

Comment:
Do you want to force npm as well? Because otherwise we wont get package lock 2.0 IIRC. 

Also we should do this for Synthetics as well ASAP, or we do synthetics first and do the lock version switch to keep things simpler. I will leave it up to you.

Reply:
My naive intention here was that node 16.15.0 comes with npm 8 so we could skip enforcing the npm version, but it is always better to be explicit 👍 Will update it.

Reply:
Thanks.